packages/ng/package.json

```diff
@@ -29,9 +29,18 @@
     "@lucca-front/icons": "*",
     "@lucca-front/scss": "*",
     "isomorphic-dompurify": "^2.17.0",
+    "@lexical/history": "^0.23.1",
+    "@lexical/link": "^0.23.1",
+    "@lexical/rich-text": "^0.23.1",
+    "@lexical/selection": "^0.23.1",
+    "@lexical/utils": "^0.23.1",
```

Comment:
Technically, all these deps are optional are not all consumer will use `@lucca-front/ng/forms/rich-text-input`